<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Transform: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pos_x">
       <property name="maximum">
        <double>530.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pos_y">
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="scale_x"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="scale_y"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rotation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rotation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>R:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="redSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>G:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="greenSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="blueSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
